### PR TITLE
Correct date pass-through on Notebook Visualizations.

### PR DESCRIPTION
**Pull request by pjfitzgibbons**
Also fix an scss error.

### Description
Notebook visualizations date pass-through from Notebook Paragraph is corrected.

### Check List
- [x] New functionality includes testing.
  - [ ] All tests pass, including unit test, integration test and doctest
- [ ] New functionality has been documented.
  - [ ] New functionality has javadoc added
  - [ ] New functionality has user manual doc added
- [x] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Review by anirudha — APPROVED** [review]

File: public/components/notebooks/components/paragraph_components/__tests__/para_output.test.tsx [hunk 1]
```diff
@@ -60,6 +60,8 @@ describe('<ParaOutput /> spec', () => {
         DashboardContainerByValueRenderer={() => null}
       />
     );
+    console.log('rendervis test', { container: utils.container.innerHTML });
```
Comment (ps48) [inline public/components/notebooks/components/paragraph_components/__tests__/para_output.test.tsx:63]: let's remove the console.log

File: public/components/notebooks/components/paragraph_components/para_output.tsx [hunk 2]
```diff
@@ -100,9 +100,9 @@ const OutputBody = ({
    */
   const dateFormat = uiSettingsService.get('dateFormat');
   const from = convertDateTime(visInput?.timeRange?.from, true, false);
-  const to = convertDateTime(visInput?.timeRange?.from, false, false);
+  const to = convertDateTime(visInput?.timeRange?.to, false, false);
   const displayFrom = convertDateTime(visInput?.timeRange?.from, true) || 'Invalid date';
-  const displayTo = convertDateTime(visInput?.timeRange?.from, false) || 'Invalid date';
+  const displayTo = convertDateTime(visInput?.timeRange?.to, false) || 'Invalid date';
```
Comment (ps48) [inline public/components/notebooks/components/paragraph_components/para_output.tsx:105]: I see the date format not being used here. We need to use the specific date-format provided by user in the dashboards settings. This helps to parse absolute dates and formats the way user wants to see it. 
2.11 implementation: https://github.com/opensearch-project/dashboards-observability/blob/920fa00b9ad3579c709f3544cb7fd319210c944c/public/components/notebooks/components/paragraph_components/para_output.tsx#L132-L135

File: public/components/event_analytics/explorer/events_views/data_grid.scss [hunk 3]
```diff
@@ -61,7 +61,7 @@
       dt {
         background-color: transparentize(shade($euiColorPrimary, 20%), 0.9);
         color: $euiTextColor;
-        padding: ($euiSizeXS / 2) $euiSizeXS;
+        padding: calc($euiSizeXS / 2) $euiSizeXS;
```
Comment (ps48) [inline public/components/event_analytics/explorer/events_views/data_grid.scss:64]: thanks 😄

File: public/components/notebooks/components/paragraph_components/__tests__/para_output.test.tsx (hunk 1 above)
Comment (pjfitzgibbons) [reply public/components/notebooks/components/paragraph_components/__tests__/para_output.test.tsx]: Yes

File: public/components/notebooks/components/paragraph_components/para_output.tsx (hunk 2 above)
Comment (pjfitzgibbons) [reply public/components/notebooks/components/paragraph_components/para_output.tsx]: Has been restored.

**Review by YANG-DB — APPROVED** [review]

**Pull request merged**